views/view-logical-workers.php

```diff
@@ -1,20 +1,26 @@
 <?php
 require("header.php"); // Important includes
-require("inc/db-magdb-open.inc.php"); // Postgres Data Sources
 
 // Configuration
+$AQUILON_URL = $CONFIG['URL']['AQUILON'];
 $config = Array(
     "clickable" => true,
 );
 
-// Gathers clusters
-$all_clusters = Array();
-$data = pg_query('select "name", "categoryName" from "vBatchAndCloudVMs"');
-if ($data and pg_num_rows($data)){
-    while ($row = pg_fetch_assoc($data)) {
-        $all_clusters[$row['name']] = $row['categoryName'];
-    }
+function is_worker_node($var) {
+    return ((strpos($var, "wn-") !== false) || (strpos($var, "hadoop-") !== false));
```

Comment:
As much as I don't like hard coding strings with conditionals, I have no alternative at this time. However, do we need the "hadoop" tranche defined? I'm in favor of removing this reference and decomming the tranches

Reply:
Yes, but not in this PR (this is just moved code).